src/nunavut/__init__.py

```diff
@@ -64,55 +64,20 @@
 """
 
 import collections
-import enum
 import pathlib
 import sys
 import typing
 
 import pydsdl
 
-from . import lang
+from .lang import LanguageContext
+from .lang._common import IncludeGenerator
 
 if sys.version_info[:2] < (3, 5):  # pragma: no cover
     print("A newer version of Python is required", file=sys.stderr)
     sys.exit(1)
 
 
-@enum.unique
-class YesNoDefault(enum.Enum):
```

Comment:
This change breaks the library API. `YesNoDefault` should be re-exported at the top level to avoid breakage.

Reply:
doh! Good catch.